### PR TITLE
WIFI-2375 Modified Dockerfile to reflect the latest changes

### PR DESCRIPTION
**Pull request by jaspreetsachdev**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,13 @@
 FROM python:3.8
-WORKDIR /ci
-
+WORKDIR /wlan-testing
 RUN apt update && apt install vim -y && rm -rf /var/lib/apt/lists/*
```
Comment (brennerm) [inline docker/Dockerfile:3]: Why do we install VIM here?

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+## Docker based environment
+
+### Building a docker image
+
+From the root directory of this repository (wlan-testing) run the following command:
+```bash
+docker build -f ./docker/Dockerfile -t wlantest
```
Comment (brennerm) [inline docker/README.md:7]: Think it should be `docker build -f ./docker/Dockerfile -t wlantest .`, right?

**Review by shivamcandela — DISMISSED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (jaspreetsachdev) [reply docker/Dockerfile:3]: Good catch. Don't think we need it. I will remove it.

File: docker/README.md (hunk 2 above)
Comment (jaspreetsachdev) [reply docker/README.md:7]: Thats right. Will make that fix.

File: docker/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 FROM python:3.8
 WORKDIR /wlan-testing
-RUN apt update && apt install vim -y && rm -rf /var/lib/apt/lists/*
+RUN apt update && rm -rf /var/lib/apt/lists/*
```
Comment (brennerm) [inline docker/Dockerfile:3]: This line can be removed completely now.

**Review by brennerm — APPROVED** [review]

**Pull request merged**